Wikipedia/Code/WMFArticleFetcher.m

```diff
@@ -80,35 +63,25 @@ - (instancetype)initWithDataStore:(MWKDataStore *)dataStore {
 }
 
 - (void)dealloc {
-    [self.operationManager invalidateSessionCancelingTasks:YES];
-    [self.pageSummarySessionManager invalidateSessionCancelingTasks:YES];
+    [self cancelAllFetches];
```

Comment:
`WMFLegacyFetcher` calls `cancelAllFetches` in its `dealloc` so I guess this could be deleted?

Reply:
yup! done